apps/portal/src/pages/index.astro

```diff
@@ -32,7 +32,7 @@ const links = [
     </style>
   </head>
 
-  <body class='iui-root'>
+  <body class='iui-root' data-iui-theme>
```

Comment:
So can `class='iui-root'` here and in the html pages be removed if it is not needed anymore?

Reply:
it is still needed for the global css. just not for variables.